### DISCUSSION TITLE
add an attribute to date_range to limit the selection of dates to within given range

### DISCUSSION THREAD

**Discussion by dennis31197** [Avaiga/taipy#954] [Ideas]
At the moment even if the provides range is for example 2024-01-01 to 2024-01-31 the control lets you select dates outside this range. This leads to extra effort for exception handling.
Would be nice to have a parameter so the user can only select dates that are within the range.

**Comment by FlorianJacta** [comment]
Thank you for your question. Indeed, this could be improved. We have a similar [issue](https://github.com/Avaiga/taipy/issues/848). Do you think it fulfills your needs?

**Reply by dennis31197** [reply]
Yes, that's what I am looking for, great.
